docs/components/copy.md

```diff
@@ -127,7 +129,13 @@ Tooltip text displayed on successful copy. Defaults to `Copied!`.
 
 ### monospace
 
-An indicator of whether the copyable text has `JetBrains Mono` font or not. Defaults to `true`, but `false` if `badge` is `true`.
+When `badge` is `true`, use this prop to control whether the copyable text has `JetBrains Mono` font or not. Defaults to `false`.
+
+<KCopy badge monospace :text="text" />
```

Comment:
When the badge is monospace, it looks like a font weight of `600` is being applied; I don't think it should be

Reply:
![image](https://github.com/Kong/kongponents/assets/2229946/0d6b978f-d093-4372-9241-15f24a7fd404)
But then on the sandbox it looks regular and the other badges look bold. Something is going on.

Reply:
Badge are supposed to be bold (to match the KBadge styles) but I agree that monospaced ones don't. Fixed now.